store/hbase/txn.go

```diff
@@ -103,6 +103,18 @@ func (txn *hbaseTxn) Get(k kv.Key) ([]byte, error) {
 	return val, nil
 }
 
+func (txn *hbaseTxn) BatchGet(keys []kv.Key) (map[string][]byte, error) {
+	encodedKeys := make([]kv.Key, len(keys))
+	for i, k := range keys {
+		encodedKeys[i] = kv.EncodeKey(k)
+	}
+	return txn.UnionStore.Snapshot.BatchGet(encodedKeys)
+}
+
```

Comment:
I suppose we should get from WBuffer first, then gets those missing keys in Snapshot.